### ISSUE TITLE
iOS SDK is initialized with wrapper sdk version 0.1.0

### ISSUE THREAD

**Issue by TroubleMakerBen** [Microsoft/mobile-center-sdk-react-native#28]
I just came across this today

```
+ (void) configureMobileCenter
{
  if (![MSMobileCenter isConfigured]) {
      MSWrapperSdk * wrapperSdk =
        [[MSWrapperSdk alloc]
            initWithWrapperSdkVersion:@"0.1.0"
            wrapperSdkName:@"mobilecenter.react-native"
            liveUpdateReleaseLabel:nil
            liveUpdateDeploymentKey:nil
            liveUpdatePackageHash:nil];
      [self setWrapperSdk:wrapperSdk];
      [MSMobileCenter configureWithAppSecret:[RNMobileCenter getAppSecret]];
  }
}
```

it should be 0.4.0, right?

**Comment by vonovak** [comment]
where did you find this?

**Comment by TroubleMakerBen** [comment]
It's in https://github.com/Microsoft/mobile-center-sdk-react-native/blob/master/RNMobileCenter/ios/RNMobileCenter/RNMobileCenter.m#L26.

@BretJohnson knows about it. =)

**Comment by BretJohnson** [comment]
The effects of this issue are benign currently, but we should fix it in the next React Native SDK update.
@max-mironov - next time we update the SDK (in a couple weeks, I expect), let's be sure to update this version number as well.  I had missed it before but just added to the release-steps doc list of version numbers to update, so we won't miss it again.  Once we publish an updated SDK, we can close this issue.

**Comment by BretJohnson** [comment]
We fixed this, in the 0.5.0 SDK update earlier this week.
